index.bs

```diff
@@ -4478,33 +4475,22 @@ entails in the ECMAScript language binding.
 <div class="example">
 
     Here is are some examples of wording to use to create and throw exceptions.
-    To throw a new [=simple exception=] named
-    {{TypeError}}:
+    To throw a new [=simple exception=] named <emu-val>TypeError</emu-val>:
 
     <blockquote>
-
-        [=Throw=] a TypeError.
-
+        [=Throw=] a <emu-val>TypeError</emu-val>.
```

Comment:
So, I think in practice we want specifications to use {{TypeError}}, not `<emu-val>TypeError</emu-val>`. The emu-val is used for the "bindings" sections of specs like Web IDL and ES. So it's a bit confusing since this is the only place in the document that we'd use {{TypeError}}, but I think it makes sense.